---
title: AI search of Neanderthal proteins resurrects ‘extinct’ antibiotics
author: Saima Sidik
date: 2023-06-28
subDate: 2023/06/28
image: https://media.nature.com/lw767/magazine-assets/d41586-023-02403-0/d41586-023-02403-0_25858106.jpg?as=webp
description: Scientists identify protein snippets made by extinct hominins.
category: Bio
tags:
  - protein
  - ai
  - antibiotics
  - dna
  - rna
---

# {{ $frontmatter.title }}

{{ $frontmatter.description }}

_[{{ $frontmatter.author }}](https://www.nature.com/search?author=Saima+Sidik)_ | {{ $frontmatter.subDate}}

![banner](https://media.nature.com/lw767/magazine-assets/d41586-023-02403-0/d41586-023-02403-0_25858106.jpg?as=webp)
_Newly identified protein snippets from Neanderthals have bacteria-fighting powers.Credit: S. Entressangle/E. Daynes/Science Photo Library_

Bioengineers have used artificial intelligence (AI) to bring molecules back from the dead [1].

To perform this molecular ‘de-extinction’, the researchers applied computational methods to data about proteins from both modern humans (Homo sapiens) and our long-extinct relatives, Neanderthals (Homo neanderthalensis) and Denisovans. This allowed the authors to identify molecules that can kill disease-causing bacteria — and that could inspire new drugs to treat human infections.

“We’re motivated by the notion of bringing back molecules from the past to address problems that we have today,” says Cesar de la Fuente, a co-author of the study and a bioengineer at the University of Pennsylvania in Philadelphia. The study was published on 28 July in Cell Host & Microbe1.

## Looking to the past

Antibiotic development has slowed over the past few decades, and most of the antibiotics prescribed today have been on the market for more than 30 years. Meanwhile, antibiotic-resistant bacteria are on the rise, so a new wave of treatments will soon be needed.

<div style="border: 1px solid #42b883; width: 250px; float: left; margin: 10px 5px 0 0; padding: 5px;">
  <a href="https://www.nature.com/articles/d41586-022-00228-x">
    <img src="https://media.nature.com/w400/magazine-assets/d41586-023-02403-0/d41586-023-02403-0_20091848.jpg" alt="benner" />
    <p style="margin-bottom: 0;">
      The staggering death toll of drug-resistant bacteria
    </p>
  </a>
</div>

Many organisms produce short protein subunits called peptides that have antimicrobial properties. A handful of antimicrobial peptides, most of which were isolated from bacteria, are already in clinical use.

The proteins of extinct species could be an untapped resource for antibiotic development — a realization to which de la Fuente and his collaborators came to thanks, in part, to a classic blockbuster. “We started actually thinking about Jurassic Park,” he says. Rather than bringing dinosaurs back to life, as scientists did in the 1993 film, the team came up with a more feasible idea: “Why not bring back molecules?”

The researchers trained an AI algorithm to recognize sites on human proteins where they are known to be cut into peptides. To find new peptides, the team applied its algorithm to publicly available protein sequences — maps of the amino acids in a protein — of H. sapiens, H. neanderthalensis and Denisovans. The researchers then used the properties of previously-described antimicrobial peptides to predict which of these new peptides might kill bacteria.

Finding and testing drug candidates using AI takes a matter of weeks. In contrast, it takes three to six years using older methods to discover a single new antibiotic, de la Fuente says.

## Ancient antibiotics

The researchers tested dozens of peptides to see whether they could kill bacteria in laboratory dishes. They then selected six potent peptides — four from H. sapiens, one from H. neanderthalensis and one from Denisovans — and gave them to mice infected with the bacterium Acinetobacter baumannii, a common cause of hospital-borne infections in humans.

<div style="border: 1px solid #42b883; width: 250px; float: left; margin: 10px 5px 0 0; padding: 5px;">
  <a href="https://www.nature.com/articles/d41586-022-02947-7">
    <img src="https://media.nature.com/w400/magazine-assets/d41586-023-02403-0/d41586-023-02403-0_25598318.gif" alt="benner" />
    <p style="margin-bottom: 0;">
      Scientists are using AI to dream up revolutionary new proteins
    </p>
  </a>
</div>

All six peptides halted the growth of A. baumannii growing in thigh muscle, but none killed the bacteria. Five of the molecules killed bacteria growing in skin abscesses, but it took a heavy hit. The doses used were “extremely high”, says Nathanael Gray, a chemical biologist at Stanford University in California.

Tweaking the most successful molecules could create more effective versions, de la Fuente says. Likewise, altering the algorithm could improve antimicrobial-peptide identification, with fewer false positives. “Even though the algorithm that we used didn’t yield amazing molecules, I think the concept and the framework represents an entirely new avenue for thinking about drug discovery,” de la Fuente says.

“The big-picture idea is interesting,” says Gray. But until the algorithm can predict clinically relevant peptides with a higher degree of success than now, he doesn’t think that molecular de-extinction will have much of an impact on drug discovery.

Euan Ashley, a genomics and precision-health expert at Stanford University in California, is excited to see a new approach in the understudied field of antibiotic development. De la Fuente and his colleagues “persuaded me that diving into the archaic human genome was an interesting and potentially useful approach”.

_[doi: https://doi.org/10.1038/d41586-023-02403-0](https://doi.org/10.1038/d41586-023-02403-0)_

## References

1. Maasch, J. R. M. A., Torres, M. D. T., Melo, M. C. R. & de la Fuente-Nunez, C. Cell Host Microbe https://doi.org/10.1016/j.chom.2023.07.001 (2023).